upload-documentation.sh

```diff
@@ -97,4 +101,71 @@ EOF
 	rm -f ${PROJECT}.json
 fi
 
+# push the docs to the github repo:
+if [[ -z "$DOCS_HIBERNATE_ORG_TARGET_DIR" ]]; then
+  DOCS_HIBERNATE_ORG_TARGET_DIR=distribution/target/repo
+fi
+DOCUMENTATION_DIRECTORY=$(realpath $DOCUMENTATION_DIRECTORY)
+mkdir -p ${DOCS_HIBERNATE_ORG_TARGET_DIR}
+pushd ${DOCS_HIBERNATE_ORG_TARGET_DIR}
+
+git config user.email ci@hibernate.org
+git config user.name Hibernate-CI
+git clone --single-branch --depth=1 git@github.com:hibernate/docs.hibernate.org.git .
```

Comment:
Actually... you might want to use `--sparse` here. Checking out the whole tree, even without history, will be _very_ slow.

Reply:
You can get some inspiration on spare checkouts here: https://github.com/hibernate/hibernate-reactive/blob/54a0116a407d22a9c19580de2ac4e0a8535c90af/release/build.gradle#L108-L120